lib/components/molecules/Accordian/Accordian.style.js

```diff
@@ -0,0 +1,46 @@
+import styled from 'styled-components';
+
+const AccordianWrapper = styled.div`
+  display: block;
+  width: 50%;
+  margin: 10px 0;
+`;
+
+const AccordianHeader = styled.div`
```

Comment:
This is not correct style of applying CSS, you can follow same BEM style in styled components too.